.github/workflows/bot_fix_style.yaml

```diff
@@ -0,0 +1,162 @@
+name: bot fix style
+
+# triggers the action when
+on:
+  issue_comment:
+    # the PR receives a comment, or a comment is edited
+    types: [created, edited]
+  pull_request_review:
+    # triggers on a review, whether or not it is accompanied by a comment
+    types: [submitted]
+  pull_request_review_comment:
+    # triggers on a review comment
+    types: [created, edited]
+
+jobs:
+  fix_style:
+    # we set some variables. The ones of the form `${{ X }}${{ Y }}` are typically not
+    # both set simultaneously: depending on the event that triggers the PR, usually only one is set
+    env:
+      AUTHOR: ${{ github.event.comment.user.login }}${{ github.event.review.user.login }}
+      COMMENT_EVENT: ${{ github.event.comment.body }}
+      COMMENT_REVIEW: ${{ github.event.review.body }}
+    name: Fix style issues from lint
+    if: github.event.issue.pull_request
+    # && (startsWith(github.event.comment.body, 'bot fix style') || contains(toJSON(github.event.comment.body), '\nbot fix style'))
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find bot fix style
+        id: bot_fix_style
+        run: |
+          COMMENT="${COMMENT_EVENT}${COMMENT_REVIEW}"
+          # we strip `\r` since line endings from GitHub contain this character
+          COMMENT="${COMMENT//$'\r'/}"
+          # for debugging, we print some information
+          printf '%s' "${COMMENT}" | hexdump -cC
```

Comment:
What is the purpose of the hexdump?

Reply:
I had it for debugging: the previous line strips `\r` and I only noticed that those were there thanks to `hexdump`.

(In case it is useful, `hexdump` produces an output where *every* character is visible, even the non-printing ones and that is its main use here.)